src/pools/selection-strategies.ts

```diff
@@ -61,7 +52,7 @@ export function findFreeWorkerBasedOnTasks<Worker> (
  * @param createAndSetupWorker `createAndSetupWorker` bounded function.
  * @param registerWorkerMessageListener `registerWorkerMessageListener` bounded function.
  * @param destroyWorker `destroyWorker` bounded function.
- * @returns The chosen one.
+ * @returns The chosen worker.
```

Comment:
:cry: I hoped I could provide an easter egg :laughing:

Reply:
lol

Reply:
I can keep it ;)

Reply:
I would like to keep it :smiley: